### ISSUE TITLE
Inconsistent current directory between running test and debugging test in workspace type multi-crate

### ISSUE THREAD

**Issue by hlavaatch** [intellij-rust/intellij-rust#5086]
## Environment

* **IntelliJ Rust plugin version:** 0.2.117.2170-193
* **Rust toolchain version:** stable-x86_64-pc-windows 1.41.1 (f3e1a954d 2020-02-24)
* **IDE name and version:** CLion 2019.3.4 
* **Operating system:** Microsoft Windows [Version 10.0.18363.719]

## Problem description

When using workspace style crate configuration, current directory the test is run in when clicking the green triangle next to line number and selecting  "Run 'Test ...'" and "Debug 'Test ..'" differ.
For Run Test, current directory is set into the inner crate, while for Debug Test it is set to outer (workspace) crate. They should both ideally be in the inner crate.
Only way to make Debug work is to fix the current directory in the "Run test" configuration and save the configuration, then clicking the green bug icon next to Configuration selector, but that is wrong.
If you click on the Debug Test next to line number triangle, duplicate configuration with wrong current dir is always generated and run in the debugger, regardless of any saved configuration.

This makes it impossible to reference test data files reliably from the tests using relative paths.

## Steps to reproduce

Create workspace structure:

outer/Cargo.toml:
```
[workspace]
members = [
"inner",
]
```

outer/inner/Cargo.toml:
```
[package]
name = "inner"
version = "0.1.0"
authors = ["hlavac"]
edition = "2018"

[dependencies]

```

outer/inner/src/main.rs:
```
fn main() {
    println!("Hello, world!");
}

#[cfg(test)]
#[test]
fn testme() {
    let dir = std::env::current_dir().unwrap();
    eprintln!("Current dir: {}",dir.display());
}
```

Open outer/Cargo.toml with CLion.
Expand the Project view and click on the outer/inner/src/main.rs

Click on the green triangle next to `fn testme()` and select "Run 'Test testme'".
Test will show it is running inside "inner" directory.

Click on the green triangle next to `fn testme()` and select "Debug 'Test testme'".
Test will show it is running inside "outer" directory.

Temporary unsaved Configuration "Test testme" generated for by both is identical. 
Command is `test --package inner --bin inner testme -- --exact`
Current directory is outer

It seems that the `--package inner` arg causes cargo to chdir into the inner package directory, while the debuger will not do it.

Ideally, the behaviour should be identical (debuger should behave like cargo test).

**Comment by hlavaatch** [comment]
[repro.zip](https://github.com/intellij-rust/intellij-rust/files/4324715/repro.zip)

**Comment by Undin** [comment]
Duplicate of #5414
Fixed in #6504
